src/lib/services/proxy-service.ts

```diff
@@ -74,12 +74,19 @@ export class ProxyService {
         const client = await this.clientForProxyToken(token);
         const definitions = client.getFeatureToggleDefinitions() || [];
 
+        const sessionId = context.sessionId || String(Math.random());
+
         return definitions
-            .filter((feature) => client.isEnabled(feature.name, context))
+            .filter((feature) =>
+                client.isEnabled(feature.name, { ...context, sessionId }),
+            )
             .map((feature) => ({
                 name: feature.name,
                 enabled: Boolean(feature.enabled),
-                variant: client.forceGetVariant(feature.name, context),
+                variant: client.getVariant(feature.name, {
```

Comment:
we don't have to use forceGetVariant anymore. Since we have sessionId we don't have to care how getVariant is implemented locally.